grafana/Dockerfile

```diff
@@ -67,7 +67,7 @@ COPY rootfs /
 
 # Health check
 HEALTHCHECK \
-    CMD curl --fail http://127.0.0.1:1337/api/health || exit 1
+    CMD curl --fail http://127.0.0.1:3000/api/health || exit 1
```

Comment:
Not his is not right, as this doesn't check the full stack.

I think I might have forgotten to commit something. I'm currently AFK, but should be back at my desk at an hour or so.

../Frenck

Reply:
No worries, feel free to close it off, the only other option I can see is then to change the nginx config.